tools/fastg2protlib/fastg2protlib-peptides.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="fastg2protlib-peptides" name="FASTG2Protlib-Peptides" version="@VERSION@">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <description>Generate a peptide FASTA library starting with a FASTG assembly file.</description>
```

Comment:
remove the dot? Maybe something more crisp/short?

Reply:
Done